### PR TITLE
Add version to the Stackdriver agent label.

### PR DESCRIPTION
**Pull request by bogdandrutu**
Fixes https://github.com/census-instrumentation/opencensus-cpp/issues/93

### CODE REVIEW

**Comment by g-easy** [comment]
This is for https://github.com/census-instrumentation/opencensus-cpp/issues/93

File: opencensus/common/version.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPENCENSUS_COMMON_VERSION_H_
+#define OPENCENSUS_COMMON_VERSION_H_
+
+// The current version of the opencensus core library. Macro to allow literal
+// string concatenation in constexpr statements.
+#define OPENCENSUS_VERSION "0.1.0-dev"
```
Comment (g-easy) [inline opencensus/common/version.h]: 0.3.0-dev

File: opencensus/common/BUILD [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# limitations under the License.
+
+load("//opencensus:copts.bzl", "DEFAULT_COPTS", "TEST_COPTS")
+
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:private"])
+
+cc_library(
+    name = "common",
```
Comment (g-easy) [inline opencensus/common/BUILD]: I'd prefer name = "version"

Comment (bogdandrutu) [reply opencensus/common/BUILD]: This follows your convention in all the packages that you have only one artifact with the name of the package. If named version probably should not be public.

File: opencensus/common/version.h (hunk 1 above)
Comment (bogdandrutu) [reply opencensus/common/version.h]: Done.

File: opencensus/common/BUILD (hunk 2 above)
Comment (g-easy) [reply opencensus/common/BUILD]: Let's keep `name = "common"` but mark it private unless and until there's a compelling reason to expose it.
 
I was hoping to leave //opencensus/common:common for some kind of utility library later. :)

Comment (bogdandrutu) [reply opencensus/common/BUILD]: Changed to version and made it package visible only.

**Pull request merged**